### PR TITLE
Hacktoberfest: Project Euler Solution Problem 77

### PR DESCRIPTION
**Pull request by breezko**
### **Describe your change:**
Added Euler Project Solution for Problem 77

* [X] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [X] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [X] This pull request is all my own work -- I have not plagiarized.
* [X] I know that pull requests will not be merged if they fail the automated tests.
* [X] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [X] All new Python files are placed inside an existing directory.
* [X] All filenames are in all lowercase characters with no spaces or dashes.
* [X] All functions and variable names follow Python naming conventions.
* [X] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by breezko** [comment]
This is part of my hacktoberfestival contribution 2020 👍

File: project_euler/problem_77/sol1.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+5 + 3 + 2
+3 + 3 + 2 + 2
+2 + 2 + 2 + 2 + 2
+
+What is the first value which can be written
+as the sum of primes in over five thousand different ways?
+"""
+
+
+def prime_sums(L=5000) -> int:
```
Comment (dhruvmanila) [inline project_euler/problem_77/sol1.py:18]: - Please change the name of the function which returns the answer to `solution()` in line with #2786 
- Use descriptive variable names as mentioned in CONTRIBUTING.md
- Use type hints and doctests

**Review by dhruvmanila — CHANGES_REQUESTED** [review]

File: project_euler/problem_77/sol1.py (hunk 1 above)
Comment (breezko) [reply project_euler/problem_77/sol1.py]: Thanks for your reply and help.

I've fixed it with 3e74825

File: project_euler/problem_77/sol1.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
+"""
+Prime summations
+Problem 77
```
Comment (dhruvmanila) [inline project_euler/problem_77/sol1.py:3]: Please add the reference link to the problem

File: project_euler/problem_77/sol1.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+5 + 3 + 2
+3 + 3 + 2 + 2
+2 + 2 + 2 + 2 + 2
+
+What is the first value which can be written
+as the sum of primes in over five thousand different ways?
+"""
+
+
+def solution(length=5000) -> int:
```
Comment (dhruvmanila) [inline project_euler/problem_77/sol1.py:19]:
```suggestion
def solution(length: int = 5000) -> int:
```

File: project_euler/problem_77/sol1.py [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    """Ways to write sum of primes
```
Comment (dhruvmanila) [inline project_euler/problem_77/sol1.py:20]:
```suggestion
    """
    Ways to write the sum of primes
```

File: project_euler/problem_77/sol1.py [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+    Keyword arguments:
+    amount -- amount of different ways (default 5000)
+
+    Passing no parameters returns the result
+    >>> solution()
+    71
```
Comment (dhruvmanila) [inline project_euler/problem_77/sol1.py:27]: Please include more `doctest`

**Review by dhruvmanila — CHANGES_REQUESTED** [review]

**Review by dhruvmanila — COMMENTED** [review]
### Announcement:

@breezko

*This message is sent to all pull requests which are **OPEN, NOT MERGED, not containing 'wontfix' label and contains 'Project Euler' or 'Project' or 'Euler' (case doesn't matter) in their title**. If this message doesn't apply to your pull request, please ignore this.*

### Message:

This is to notify all the users submitting a pull request for the Project Euler solution that the [README.md](https://github.com/TheAlgorithms/Python/blob/master/project_euler/README.md) file has been updated with all the style requirements necessary for the directory and solution files. ***Please read through it and make all the necessary changes required to your submission.***

**Comment by GantaChaitra** [comment]
Hey @dhruvmanila ..  i would like to contribute to this issue .. can I ?

**Pull request closed without merging**